### PR TITLE
Fix 1625 1620

### PR DESCRIPTION
**Pull request by ismailsunni**
For #1620 and #1625

### CODE REVIEW

**Comment by ismailsunni** [comment]
hi @lucernae would you like to review my PR :)

File: safe/impact_functions/core.py [hunk 1]
```diff
@@ -674,8 +658,9 @@ def get_admissible_plugins(keywords=None):  # , name=None):
 
 
 def parse_single_requirement(requirement):
-    '''Parse single requirement from impact function's doc to category,
-        subcategory, layertype, datatype, unit, and disabled.'''
+    """Parse single requirement from impact function's doc to category,
```
Comment (akbargumbira) [inline safe/impact_functions/core.py]: It's better to have a one-line explanation of the function and explain more after that as a different paragraph

**Comment by akbargumbira** [comment]
LGTM

File: safe/impact_functions/core.py (hunk 1 above)
Comment (ismailsunni) [inline safe/impact_functions/core.py]: Yes, of course.

I just change the single quote to double quote. We will remove this function soon we remove the docstring. It's 80% finished. so, I will just leave it now for now

**Pull request merged**